apps/frontend/src/components/proposal/ProposalParticipants.tsx

```diff
@@ -79,41 +122,78 @@ const Participants = ({
         >
           {title}
         </Typography>
-        <PeopleTable
-          selection={false}
-          mtOptions={{
-            showTitle: false,
-            toolbar: false,
-            paging: false,
-            headerStyle: {
-              padding: '4px 10px',
-            },
-          }}
-          isFreeAction={true}
-          data={users}
-          search={false}
-          userRole={UserRole.USER}
-          invitationUserRole={UserRole.USER}
-          onRemove={removeUser}
-          preserveSelf={preserveSelf}
-          setPrincipalInvestigator={setPrincipalInvestigator}
-        />
-        <ActionButtonContainer
-          sx={(theme) => ({
-            marginTop: theme.spacing(1),
-          })}
-        >
-          <Button
-            variant="outlined"
-            onClick={openModal}
-            data-cy="add-participant-button"
-            size="small"
-            startIcon={<PersonAddIcon />}
-            disabled={loadingPrincipalInvestigator}
+        <div>
+          <PeopleTable
+            selection={false}
+            mtOptions={{
+              showTitle: false,
+              toolbar: false,
+              paging: false,
+              headerStyle: {
+                padding: '4px 10px',
+              },
+            }}
+            isFreeAction={true}
+            data={users}
+            search={false}
+            userRole={UserRole.USER}
+            invitationUserRole={UserRole.USER}
+            onRemove={removeUser}
+            preserveSelf={preserveSelf}
+            setPrincipalInvestigator={setPrincipalInvestigator}
+          />
+
+          {invites.length > 0 ? (
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'row',
+                marginTop: 1,
+                gap: 1,
+                alignItems: 'center',
+              }}
+              data-cy="invites-chips"
+            >
+              <Typography
+                sx={{
+                  fontSize: '12px',
+                  color: 'grey',
+                  paddingRight: '10px',
+                  display: 'inline-block',
+                }}
+              >
+                Invited:
+              </Typography>
+              {invites.map((invite) => (
+                <Chip
+                  sx={{ gap: '2px', padding: '6px' }}
+                  color="secondary"
+                  icon={invite.isEmailSent ? <SendIcon /> : <ScheduleSend />}
+                  size="small"
+                  label={invite.email}
+                  key={invite.email}
+                  onDelete={() => handleDeleteInvite(invite)}
+                />
+              ))}
+            </Box>
+          ) : null}
```

Comment:
Maybe we can use the short hand && rather than returning null.

Reply:
Thanks